docs/analysis_software_MRI.md

```diff
@@ -117,3 +117,14 @@
 -   paper DOI:
 -   RRID:
 -   contact: [mailing list](https://www.fil.ion.ucl.ac.uk/spm/support/)
+
+## 	GingerALE
+-   [GingerAle](http://www.brainmap.org/ale/) -   A software for meta-analysis for fMRI studies.
+-   repository URL:
+-   website URL: [http://www.brainmap.org/ale/](http://www.brainmap.org/ale/)
+-   tutorial URL: [http://www.fil.ion.ucl.ac.uk/spm/course/](http://www.fil.ion.ucl.ac.uk/spm/course/)
+-   documentation: [http://www.brainmap.org/ale/manual.pdf](http://www.brainmap.org/ale/manual.pdf)
```

Comment:
For URL, it is recommended to use this kind of formatting:
```suggestion
-   [website](http://www.brainmap.org/ale/)
-   [documentation](http://www.fil.ion.ucl.ac.uk/spm/course/)
-   [documentation](http://www.brainmap.org/ale/manual.pdf)
```

```
-   website URL: [http://www.brainmap.org/ale/](http://www.brainmap.org/ale/)
-   tutorial URL: [http://www.fil.ion.ucl.ac.uk/spm/course/](http://www.fil.ion.ucl.ac.uk/spm/course/)
-   documentation: [http://www.brainmap.org/ale/manual.pdf](http://www.brainmap.org/ale/manual.pdf)
```
renders:
-   website URL: [http://www.brainmap.org/ale/](http://www.brainmap.org/ale/)
-   tutorial URL: [http://www.fil.ion.ucl.ac.uk/spm/course/](http://www.fil.ion.ucl.ac.uk/spm/course/)
-   documentation: [http://www.brainmap.org/ale/manual.pdf](http://www.brainmap.org/ale/manual.pdf)

while
```
-   [website](http://www.brainmap.org/ale/)
-   [documentation](http://www.fil.ion.ucl.ac.uk/spm/course/)
-   [documentation](http://www.brainmap.org/ale/manual.pdf)
```
renders:
-   [website](http://www.brainmap.org/ale/)
-   [documentation](http://www.fil.ion.ucl.ac.uk/spm/course/)
-   [documentation](http://www.brainmap.org/ale/manual.pdf)